src/anubhav/assignment8/org/Main.java

```diff
@@ -0,0 +1,48 @@
+package anubhav.assignment8.org;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Exceptionthrough exp=new Exceptionthrough();
+        Scanner sc=new Scanner(System.in);
+        String int_var;
```

Comment:
the variable names should follow camelCase convention

Reply:
Actually variable name should be small I have read from java docs.
link: https://docs.google.com/document/d/1e272pOEpgCCjH5rtaoIMYouUrGFYZ4YkxiIhMOycXoY/edit